.github/workflows/android_arm64.yml

```diff
@@ -27,7 +27,7 @@ concurrency:
 
 jobs:
   Build:
-    timeout-minutes: 15
+    timeout-minutes: 40
```

Comment:
does it assume that build without cache can also pass here?
I mean the worst case when everything needs to be rebuilt.

Reply:
I am unsure. I found the longest one with 1h of build time: https://github.com/openvinotoolkit/openvino/actions/runs/6623899432/job/17991799138. Strangely enough, the `cache-hit-rate` is ~97% as indicated by logs:
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/1c05ab54-c52c-4aed-8c24-f77c99510353)

Reply:
The recent pipelines are in around the 10-minute mark (when they do not try to exceed the 15-minute timeout).